script.js

```diff
@@ -1,4 +1,275 @@
+var player1DiceRoll1 = 0;
+var player1DiceRoll2 = 0;
+var player2DiceRoll1 = 0;
+var player2DiceRoll2 = 0;
+var player1FinalDiceRollResult = 0;
+var player2FinalDiceRollResult = 0;
+var finalPlayer1AdvancedResults = 0;
+var finalPlayer2AdvancedResults = 0;
+var player1AdvanceRoll = [];
+var player2AdvanceRoll = [];
+var myOutputValue = "";
+var player1WinCount = 0;
+var player2WinCount = 0;
+var currentGameMode = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  //Output Gamemode choices "normal" and "advanced"
+  if (currentGameMode == "") {
+    myOutputValue =
+      "Please choose between the two gamemodes. Submit 'normal' for normal gamemode (i.e., 2 dices), and 'advanced' for advanced gamemode! (i.e., select no. of dices to be played, autochoice)";
+    currentGameMode = "Gamemode selected";
+    return myOutputValue;
```

Comment:
When coding we try to keep our code DRY (Don't repeat yourself). So instead of return myOutputValue in each if/else if condition, you can consider just writing it once between lines 90 and 91.